app/src/routes/constants.js

```diff
@@ -73,6 +74,7 @@ import { OrganizationUsersPage } from 'pages/organization/organizationUsersPage'
 import { OrganizationsPage } from 'pages/instance/organizationsPage';
 import { ProductVersionsPage } from 'pages/inside/productVersionsPage/productVersionsPage';
 import { TestCaseLibraryPage } from 'pages/inside/testCaseLibraryPage';
+import { TestCaseDetailsPage } from 'pages/inside/testCaseLibraryPage/testCaseDetailsPage';
```

Comment:
Let's combine since they share same parent folder

```suggestion
import { TestCaseLibraryPage, TestCaseDetailsPage } from 'pages/inside/testCaseLibraryPage';
```